Sources/LiveViewNativeStylesheet/Parsing/External/Metadata.swift

```diff
@@ -4,42 +4,56 @@ public struct Metadata {
     public let file: String
     public let line: Int
     public let module: String
+    public let source: String
     
     public static func parser() -> some Parser<Substring.UTF8View, Self> {
-        Parse { (file, line, module) in
-            Self(file: file, line: line, module: module)
-        } with: {
-            "[".utf8
-            
-            Whitespace()
-            "file:".utf8
-            Whitespace()
-            StringLiteral()
-            Whitespace()
-            ",".utf8
-            Whitespace()
-            
-            Whitespace()
-            "line:".utf8
-            Whitespace()
-            Parsers.IntParser<Substring.UTF8View, Int>()
-            Whitespace()
-            ",".utf8
-            Whitespace()
-            
-            Whitespace()
-            "module:".utf8
-            Whitespace()
-            Many {
-                Identifier()
-            } separator: {
-                ".".utf8
+        OneOf {
+            Parse { (file, line, module, source) in
+                Self(file: file, line: line, module: module, source: source)
+            } with: {
+                "[".utf8
+                
+                Whitespace()
+                "file:".utf8
+                Whitespace()
+                StringLiteral()
+                Whitespace()
+                ",".utf8
+                
+                Whitespace()
+                
+                "line:".utf8
+                Whitespace()
+                Parsers.IntParser<Substring.UTF8View, Int>()
+                Whitespace()
+                ",".utf8
+                
+                Whitespace()
+                
+                "module:".utf8
+                Whitespace()
+                Many {
+                    Identifier()
+                } separator: {
+                    ".".utf8
+                }
+                .map({ $0.joined(separator: ".") })
+                Whitespace()
+                ",".utf8
+                
+                Whitespace()
+                
+                "source:".utf8
+                Whitespace()
+                StringLiteral()
+                
+                Whitespace()
+                
+                "]".utf8
+            }
+            "[]".utf8.map {
+                Self.init(file: "", line: 0, module: "", source: "")
             }
```

Comment:
@bcardarella mentioned that production builds pass an empty metadata component. This should account for that, but let me know if there is any other data passed besides an empty list literal `[]`.

Reply:
This is correct for the modifier metadata.